plugins/module_utils/doc_fragments/ipv6_range_info.py

```diff
@@ -3,7 +3,7 @@
 specdoc_examples = ['''
 - name: Get info about an IPv6 range
   linode.cloud.ipv6_range_info:
-    range: 2600:3c01::''']
+    range: "2600:3c01::"''']
```

Comment:
It looks like this is causing the CI to fail — `make deps && make gendocs` should do the trick 🙂